index.html

```diff
@@ -119,36 +119,39 @@
         				  for edges that reflects their target terminal.
                 */
 
+
+
                 graph.model.getStyle = function(cell) {
                     var style = null;
                     if (cell != null) {
-
                         // Get style for the recently created mxCell.
                         style = mxGraphModel.prototype.getStyle.apply(this, arguments);
-
                         // If the mxCell is an edge and if it's a fully formed edge
                         if (this.isEdge(cell) && cell.source != null) {
                             var target = this.getTerminal(cell, false);
-
                             if (target != null) {
-
                                 /* cell.name attribute defines the link name
                                    so that it can be parsed in the XML during
                                    XSLT transformation.
                                 */
-
                                 var cellSource = cell.source;
-                                while (cellSource.isEdge() == true) {
-                                    cellSource = cellSource.source;
+                                while(cellSource.isEdge() == true) {
+                                  cellSource = cellSource.source;
                                 }
                                 if (cellSource.value == "ExplicitOutputPort" || cellSource.value == "ExplicitInputPort") {
-                                    style += ';' + 'ExplicitLink';
+                                    if(style == null) {
+                                      style = ';' + 'ExplicitLink' + ';';
```

Comment:
The code is uncommented and looks messy now, I'll fix all of this first thing tomorrow.